### PR TITLE
[DNM] Update ATI contentId logic after Ares update

### LINKED ISSUE #9607: Update ATI ContentId logic for CPS and Optimo assets
**Is your feature request related to a problem? Please describe.**
We currently construct the ContentId in Simorgh.  Ares now provides the value for us.  We can simplify Simorgh by using the value provided by Ares.

**Describe the solution you'd like**
Ares have updated their output for both CPS and Optimo assets to explicitly expose the ContentId value to send to ATI.

We should remove the existing logic in Simorgh which generates this, and replace the value with the one provided by Ares as metadata.analyticsLabels.contentId.  This should apply to both AMP and Canonical.

Current Optimo implementation: https://github.com/bbc/simorgh/blob/538a49c1b7368c708e4ddfd61c31d44db63f9f5b/src/app/lib/analyticsUtils/article/index.js#L6-L13
Current CPS implementation: https://github.com/bbc/simorgh/blob/538a49c1b7368c708e4ddfd61c31d44db63f9f5b/src/app/lib/analyticsUtils/index.js#L223-L228

This change means we no longer need separate treatment for CPS and Optimo asset here, nor are we constructing the value for CPS assets.

Note that for CPS assets this will result in "curie:asset:" being added additionally in the content id, compared to what we have now. This is expected, and standardises the contentId value with that used in other systems such as Passports and SCV.

Similarly, for Optimo this will result in "asset:" being added additionally in the content id, compared to what we have now.  This is also expected, for the same reasons.

# Examples
## Optimo

Ares: https://ares-api.api.bbci.co.uk/api/article/c1ddvpj8zkro
FABL: https://fabl.api.bbci.co.uk/module/article?urn=urn:bbc:optimo:asset:c1ddvpj8zkro

### Ares data
```
{
  "metadata": {
    "analyticsLabels": {
      "contentId": "urn:bbc:optimo:asset:c1ddvpj8zkro"
```
## CPS

Ares: https://ares-api.api.bbci.co.uk/api/asset/news/science-environment-59022676
FABL: https://fabl.api.bbci.co.uk/module/article?uri=/news/science-environment-59022676
### Ares data
```
{
  "metadata": {
    "analyticsLabels": {
      "contentId": "urn:bbc:cps:curie:asset:29375628-9511-42e6-be88-ebaa1158f597",
```

**Describe alternatives you've considered**
Update logic in Simorgh to construct URNs matching those already provided by Ares.  This seems pointless duplication and fragile.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves #9607

**Overall change:**
This PR removes the logic used to construct the contentId for CPS and Optimo articles, as Ares now provide the value for us.

**Code changes:**

- Removed logic that constructed the contentId.
- Updated code to use the contentId provided by Ares
- Updated tests and fixture data to reflect change

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by aeroplanejane** [comment]
@rebeccamcginn can we hold on merging this. @jimjohnsonrollings has spotted an issue caused by this change on PS webcore. Will update as I learn more.

**Comment by DarioR01** [comment]
Can this be reviewed in the meanwhile?

**Comment by jimjohnsonrollings** [comment]
Fine for it to be reviewed.  I think releasing the CPS side will be blocked for a while now. Releasing the Optimo side however is not blocked... so might be worth considering splitting into separate PRs :(

File: src/app/contexts/EventTrackingContext/index.test.jsx [hunk 1]
```diff
@@ -136,11 +136,6 @@ describe('Error handling', () => {
 
     expect(trackingData).toEqual({});
     expect(errorMessage).toBeUndefined();
-    expect(console.error).toHaveBeenCalledWith(
```
Comment (jroebu14) [inline src/app/contexts/EventTrackingContext/index.test.jsx:139]: why was this removed?

Comment (rebeccamcginn) [reply src/app/contexts/EventTrackingContext/index.test.jsx:139]: When we used the getContentId function, we attempted to call split on curie: https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.js#L232

In this test, we don't provide curie and an error is thrown, caught here: https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/params/index.js#L123 

I've removed the use of split and replaced it with path, so we no longer see the error expected in this assertion.
